pkg/acceptance/cluster/dockercluster.go

```diff
@@ -517,21 +519,25 @@ func (l *DockerCluster) startNode(ctx context.Context, node *testNode) {
 // automatically, but exposed for tests that use INIT_NONE. nodeIdx
 // may designate any node in the cluster as the target of the command.
 func (l *DockerCluster) RunInitCommand(ctx context.Context, nodeIdx int) {
+	// Add a randomID to the container name to avoid overlap between tests running on
+	// different shards.
+	rnd := rand.New(rand.NewSource(timeutil.Now().UnixNano()))
```

Comment:
Nit: It's not impossible for two shards to start at the same nanosecond, or for two to get to this point at the same nanosecond. Maybe use some cryptographic randomness or something if possible?

Reply:
Updated